app/src/components/tables/BlockTable/BlocksTable.tsx

```diff
@@ -250,36 +250,39 @@ export const BlocksTable: React.FC = () => {
       sorting={[
         {
           id: blocksTableOptions.sorting.sortBy,
-          desc: blocksTableOptions.sorting.order === 'desc',
+          desc: blocksTableOptions.sorting.order === TableOrder.Descending,
         },
       ]}
       onSortingChange={() => {
         setIsTableLoading(true);
         dispatch(
           updateBlocksSorting({
             sortBy: 'height',
-            order: blocksTableOptions.sorting.order === 'desc' ? 'asc' : 'desc',
+            order:
+              blocksTableOptions.sorting.order === TableOrder.Descending
+                ? TableOrder.Ascending
+                : TableOrder.Descending,
           }),
         );
       }}
       initialSorting={initialSorting}
     />
   );
 };
-const BlocksTableHead = styled.div`
+export const BlocksTableHead = styled.div`
```

Comment:
Do we need to export all of these?

Reply:
removed the unused exports

Reply:
Can you explain why you're exporting them?

Reply:
I opted to leave them out of the test file to reduce clutter. Anything that was over a few lines and constant I threw into the mock data file.